Remove [bumpversion:file:setup.py] from setup.cfg

Otherwise

```bash
$ bumpversion minor

Traceback (most recent call last):
  File "/usr/local/bin/bumpversion", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/bumpversion/__init__.py", line 912, in main
    f.should_contain_version(current_version, context)
  File "/usr/local/lib/python2.7/dist-packages/bumpversion/__init__.py", line 208, in should_contain_version
    assert self.contains(version.original), msg
AssertionError: Did not find '0.1.0' or '0.1.0' in file setup.py
```

since

```python
# setup.py
#...
version = get_version("dp", "__init__.py")
#...
```

--- a/{{cookiecutter.repo_name}}/setup.cfg
+++ b/{{cookiecutter.repo_name}}/setup.cfg
@@ -3,8 +3,6 @@ current_version = {{ cookiecutter.version }}
 commit = True
 tag = True
 
-[bumpversion:file:setup.py]
-
 [bumpversion:file:{{ cookiecutter.app_name }}/__init__.py]
 
 [wheel]

